### PR TITLE
Add lints and update CHANGELOG.md

### PR DESCRIPTION
**Pull request by vilyapilya**
add lint
change the regex to semver
update the changelog.md file

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -43,17 +52,23 @@ workflows:
...
   update:
     jobs:
+     - eslint:
+        filters:
+          tags:
+            only: /[0-9].[0-9].[0-9]/
+          branches:
+            ignore: /.*/
       - build:
           filters:
```
Comment (smusali) [inline .circleci/config.yml]: you should add the following before `filters`:
```
requires:
  - test
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,18 +2,21 @@
 
 This file documents all notable changes in `LogDNA CloudWatch Lambda Function`. The release numbering uses [semantic versioning](http://semver.org).
 
+## v2.0.2, v2.0.3 - Released on November 1, 2019
```
Comment (smusali) [inline CHANGELOG.md]: I think I have released `v2.0.3` not on the same day

it's okay to have separate empty section for that as following:
```
## v2.0.3 - Released on November .., 2019
```

File: .circleci/config.yml (hunk 1 above)
Comment (vilyapilya) [reply .circleci/config.yml]: you mean -eslint?

Comment (smusali) [reply .circleci/config.yml]: yup

**Comment by smusali** [comment]
Things are getting handled [here](https://github.com/logdna/logdna-cloudwatch/pull/18)

**Pull request closed without merging**